### ISSUE TITLE
InvalidArgumentError: Incompatible shapes: [2,256,256,1] vs. [0,256,256,1]

### ISSUE THREAD

**Issue by Muskaan06** [zhixuhao/unet#227]
Found 20 images belonging to 1 classes.
Found 0 images belonging to 1 classes.

My images folder are being read but the mask folder is not. Please help.

**Comment by Mullefar1** [comment]
Did you ensure, that mask/label folder is placed correct? For the standard code, you need to have the image folder and mask/label folder under the same folder. My path looks like this: 

Generator = train(8, 'My folder with train data', 'image','label', data_gen_args, save_to_dir = None)

**Comment by Muskaan06** [comment]
yes the folder name was okay.....my train file contains .png images and this model reads only .tif. It's working when I changed the type but not getting the result.
